### PR TITLE
BRADD added

### PR DESCRIPTION
**Pull request by dug9**
BRADD - blender to radiance direct

### CODE REVIEW

File: bradd/bradd_wiki/wiki_bradd27_blend279.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
```
Comment (pisuke) [inline bradd/bradd_wiki/wiki_bradd27_blend279.html:1]: Why not adding the documentation directly on the [wiki](https://github.com/pisuke/brad/wiki)?

File: bradd/bradd_wiki/wiki_bradd27_blend279.html [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    <tr>
+      <td>-rotation</td>
+      <td>-rx -ry -rz</td>
+    </tr>
+    <tr>
+      <td>-scale</td>
+      <td>-s SX only applied, assumes iso-tropic (if an-isotropic scale, please Object > Apply > Scale before export</td>
+    </tr>
+    <tr>
+      <td>nested heirarchy</td>
```
Comment (pisuke) [inline bradd/bradd_wiki/wiki_bradd27_blend279.html]: Small typo: hierarchy

**Review by pisuke — COMMENTED** [review]
Thank you @dug9 for the contribution!
Only thing to discuss before merging is whether it's best to put the documentation HTML page directly to the wiki. Happy to do that if you like the idea. Thanks!

**Comment by dug9** [comment]
Fransesco - thanks for the detailed review! yes sure - I don't know where the wiki is - could you do it for me?
Thanks,
-Doug

________________________________________

**Comment by pisuke** [comment]
OK, I'll do the wiki and some refactoring when I have a moment. 
In the meanwhile, I'll merge your code.
Thanks for the contribution!

**Comment by dug9** [comment]
And thank you Francesco for your efforts.
-Doug
I'll post on discourse saying its over here now.

PS I can see my http link strategy was wrong when the html is on GitHub, which forces to plain/text according to:
https://github.com/htmlpreview/htmlpreview.github.com
So what I should have done in they python wiki url to link to the html on GitHub, is prefix with
https://htmlpreview.github.io/?
like this:
https://htmlpreview.github.io/?https://github.com/pisuke/brad/blob/master/bradd/bradd_wiki/wiki_bradd27_blend279.html




________________________________________


OK, I'll do the wiki and some refactoring when I have a moment.
In the meanwhile, I'll merge your code.
Thanks for the contribution!

**Pull request merged**